hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/DeletingServiceMetrics.java

```diff
@@ -160,6 +201,92 @@ public void incrNumRenameEntriesPurged(long renameEntriesPurged) {
     this.numRenameEntriesPurged.incr(renameEntriesPurged);
   }
 
+  public void setKdsLastRunTimestamp(long timestamp) {
+    this.kdsLastRunTimestamp.set(timestamp);
+  }
+
+  public void setKdsCurRunTimestamp(long timestamp) {
+    this.kdsCurRunTimestamp.set(timestamp);
+  }
+
+  private void resetMetrics() {
```

Comment:
why reset metrics after 24hour? usually metrics are reset when restart, and that's good enough.

Reply:
We already collect deletion metrics since OM uptime. The thought was on OM UI it showing the progress of deletion over the last 24h would be more interesting for someone who has performed a delete and wants to track how the delete is proceeding.